mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -49,93 +120,282 @@ impl mayastor_server::Mayastor for MayastorSvc {
             return Err(Status::invalid_argument("Missing devices"));
         }
 
-        sync_config(pool_grpc::create(args)).await
+        let rx = rpc_submit::<_, _, LvsError>(async move {
```

Comment:
Do we need to update the config file on pool create/destroy operations?

Reply:
Yes, in fact, we only need to go over this for "pools" but that is covered in a separate ticket. I'm just doing half the job already =)

Reply:
Ah ok, though I'd recommend not merging this one until the other ticket's PR comes around otherwise develop will not work with pod restarts in k8s